components/Alert/index.html

```diff
@@ -78,10 +78,77 @@ <h3 class="sidebar-head">Components</h3>
 					</li>
 				</ul>
 			</div>
-			<div class="component">
-				<h2>Alert</h2>
+			<div class="flex-center-component">
+				<div class="component">
+					<h1 class="component-header">Alert</h1>
+					<p class="component-text">
+						Alerts are used to attract user's attention for important
+						information without interrupting the user's flow. Use the class
+						<span class="highlight">alert</span> for alert component and classes
+						<span class="highlight">
+							alert-info alert-success alert-error alert-warning</span
+						>
+						for alerts of different types.
+					</p>
+
+					<div class="component-display vertical">
+						<div class="alert alert-info">
+							<div class="alert-msg">
+								<span class="alert-icon"
+									><i class="fas fa-info-circle"></i></span
+								>This is an info alert!
+							</div>
+							<div class="alert-close-btn">
+								<i class="fas fa-times"></i>
+							</div>
+						</div>
+						<div class="alert alert-success">
+							<div class="alert-msg">
+								<span class="alert-icon"
+									><i class="fas fa-check-circle"></i></span
+								>This is a success alert!
+							</div>
+							<div class="alert-close-btn">
+								<i class="fas fa-times"></i>
+							</div>
+						</div>
+						<div class="alert alert-error">
+							<div class="alert-msg">
+								<span class="alert-icon"
+									><i class="fas fa-exclamation-circle"></i></span
+								>This is an error alert!
+							</div>
+							<div class="alert-close-btn">
+								<i class="fas fa-times"></i>
+							</div>
+						</div>
+						<div class="alert alert-warning">
+							<div class="alert-msg">
```

Comment:
can reduce additional div here and make changes to the parent div itself, in an alert components flex is mostly row so I don't think any problem will be there

Reply:
Hmm. actually, the alert message is on the left and the X button is on the right, so I had to use a div with flex for that. And the message contains 2 parts(icon + text) so I had to use a div for it. So I'm not sure which div I can emit. The one I can consider emitting out is Probably the div for the X button. Is this what you are referring to?